Tests/FluentAssertions.Specs/Numeric/NumericAssertionSpecs.cs

```diff
@@ -38,7 +38,7 @@ public void When_a_negative_value_is_positive_it_should_throw()
             public void When_a_zero_value_is_positive_it_should_throw()
             {
                 // Arrange
-                int value = 0;
+                double value = double.NaN;
```

Comment:
Was this change intentional?
The method name says "zero value", but body uses NaN

Reply:
Good catch. Must have been experimenting with something.